setup.py

```diff
@@ -55,13 +53,11 @@
         'Topic :: Text Processing',
         'Topic :: Text Processing :: Fonts'],
     install_requires=[
-        'booleanOperations',
-        'flat',
-        'fontTools',
-        'libsass',
-        'markdown',
-        'pyobjc; platform_system=="darwin"',
-        'SimpleIDML',
-        'svgwrite',
-        'tornado']
+        'pagebot',
+        # Dependencies that are not yet available on PyPI:
```

Comment:
no, don't use this `@` links in here. In setup.py `install_requires` keep just the abstract deps (i.e. with their name and possibly a minimum version required). Then in `requirements.txt` file you write your concrete dependencies (with exact version specifiers, and if not on PyPI yet, with the git url from where to download the source), and do `pip install -r requirements.txt` to initialize or sync your venv.

E.g. to install "drawbot" from the Github source you can write this in your `requirements.txt` file:
(see this [pip docs](https://pip.pypa.io/en/latest/reference/pip_install/#git) page if you want to install from different branch or specific tag or commit)

```

-e git+https://github.com/typemytype/drawbot.git#egg=drawbot
```